features/Project_Urgency.feature

```diff
@@ -4,102 +4,77 @@ Feature: Create a new project application for Project Urgency
   I want to submit a project urgeny proposal for flood and coastal erosion risk management project funding
   So that my local area can have improved protection from flooding
 
+  Background: 
+    Given I am an external user
+    And I have a valid "rma" username and password
+    And I create a new proposal
+    And I enter a project name
+
   @regression @projecturgency @RMAUser
   Scenario: Submit a new proposal with not urgent project urgency for rma user
-    Given I am an external user
-      And I have a valid "rma" username and password
-      And I create a new proposal
-      And I enter a project name
-      And I select a project type "change_or_new_asset"
+    Given I select a project type "change_or_new_asset"
       And I select financial year to stop spending
     When I select the project urgency as "not_urgent"
     Then I should see the project urgent message of "Your project is not urgent" on the proposal page
```

Comment:
Seeing a few places where the alignment of these scenario steps are not consistent. Would be worth opening a PR to tidy this up after this is merged.

Reply:
Fixed in the next commit where the And should be under the Given, When or Then as to the example below

  Scenario: User requests a sell before close of trading
    Given I have 100 shares of MSFT stock
       And I have 150 shares of APPL stock
       And the time is before close of trading